ts/features/bonus/cgn/saga/networking/categories/cgnCategoriesSaga.ts

```diff
@@ -7,6 +7,15 @@ import {
   getNetworkError
 } from "../../../../../../utils/errors";
 import { BackendCgnMerchants } from "../../../api/backendCgnMerchants";
+import { PublishedProductCategoriesWithNewDiscountsCount } from "../../../../../../../definitions/cgn/merchants/PublishedProductCategoriesWithNewDiscountsCount";
+import { PublishedProductCategories } from "../../../../../../../definitions/cgn/merchants/PublishedProductCategories";
+
+const checkIsCategoriesWithCount = (
+  cl:
+    | PublishedProductCategoriesWithNewDiscountsCount
+    | PublishedProductCategories
+): cl is PublishedProductCategoriesWithNewDiscountsCount =>
+  PublishedProductCategoriesWithNewDiscountsCount.decode(cl).isRight();
```

Comment:
```suggestion
  PublishedProductCategoriesWithNewDiscountsCount.is(cl);
```
the same but more idiomatic